go-controller/pkg/clustermanager/egressip_controller_test.go

```diff
@@ -4049,11 +4208,32 @@ var _ = ginkgo.Describe("OVN cluster-manager EgressIP Operations", func() {
 				gomega.Eventually(getEgressIP).Should(gomega.Equal(updateEgressIP))
 				_, nodes = getEgressIPStatus(egressIPName)
 				gomega.Expect(nodes[0]).To(gomega.Equal(egressNode2.name))
+				markAfterUpdate := getEgressIPAnnotationValue(egressIPName, util.EgressIPMarkAnnotation)()
+				gomega.Expect(markBeforeUpdate).Should(gomega.Equal(markAfterUpdate), "The mark should be consistent")
+				markInCache, err := fakeClusterManagerOVN.eIPC.getOrAllocMark(egressIPName)
+				gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
+				gomega.Expect(markAfterUpdate).Should(gomega.Equal(fmt.Sprintf("%d", markInCache)))
 				return nil
 			}
 
 			err := app.Run([]string{app.Name})
 			gomega.Expect(err).NotTo(gomega.HaveOccurred())
 		})
 	})
+
+	ginkgo.Context("EgressIP Mark cache", func() {
```

Comment:
I wrote one and tested it but it took a long time to iterate through all the range values - something like 2 minutes on my local PC. I actually dont want it to return an error if range is exhausted because i dont want to block EIP assignment for pods assigned to CDN. I tested to make sure it doesnt return an error for that scenario.

Reply:
Can't you have a way to inject a max value to the range for the tests ? (really asking)